src/Link/Link.js

```diff
@@ -76,7 +76,7 @@ const linkStyles = ({ variant, theme: baseTheme }) => {
 };
 
 /**
- * The Link component creates a hyperlink to another URL.
+ * The Link component creates a hyperlink to another URI.
```

Comment:
Whatever we use here should match the prop doc.

Reply:
changed to URL